### PR TITLE
Drugs at FDA Import Revamp

### PR DESCRIPTION
**Pull request by clincoln8**
Reformatted FDA import. Broken into two parts - cleaning + generating mcf. Only takes ~5min to run by storing the drug name to ChEMBL relationship in utils/drug_refs.json, instead of querying ChEMBL for every drug every time the scripts are run. 
All dictionaries and constants are in utils/config.py.

### CODE REVIEW

**Review by spiekos — COMMENTED** [review]
Hey Christie! It looks good overall. Just some minor changes, mostly typos, which won't require rerunning everything after they're made.

File: scripts/us_fda/drugs/clean.py [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+
+    Takes in dictionaries containing in Marketing Status information and
+    Therapeutic Equivalence information to append their data to the DataFrame.
+    Cleans the raw data from the .txt files and creates new columns based on
+    extra within the original columns.
+    """
+
+    #progress PRINSTON
+    if row.name % 5000 == 0:
+        print('...' + str(row.name) + '/' + str(df_length) + ' rows expanded')
```
[the remark this answers is not in the document]
Comment (clincoln8) [reply scripts/us_fda/drugs/clean.py:354]: line 429 prints "...expanding DataFrame', then the function expand_df is called in a loop so there should be some more context for the user when clean.py is running. Is this enough, or should more be added to the "...expanding DataFrame" statement?

[the remark this answers is not in the document]
Comment (spiekos) [reply scripts/us_fda/drugs/clean.py:354]: That's enough context. That sounds good.

**Review by spiekos — APPROVED** [review]

**Comment by spiekos** [comment]
\gcbrun

**Pull request merged**